components/cdc/src/delegate.rs

```diff
@@ -887,6 +975,30 @@ fn decode_lock(key: Vec<u8>, lock: Lock, row: &mut EventRow) -> bool {
     false
 }
 
+fn decode_rawkv(key: Vec<u8>, value: Vec<u8>, row: &mut EventRow, is_ignore_delete: bool) -> bool {
+    let decoded_value = APIV2::decode_raw_value(&value).unwrap();
+    if decoded_value.is_delete && is_ignore_delete {
```

Comment:
Logical deleted entry is needed as long as it is in the timestamp range of request, both in incremental scan & cdc.